bin/node/src/chain_spec.rs

```diff
@@ -60,6 +94,11 @@ pub struct ChainParams {
     #[structopt(long, short = "d", value_name = "PATH", parse(from_os_str))]
     pub base_path: PathBuf,
 
+    /// Specify filename to write node private p2p keys to
+    /// Resulting keys will be stored at: base_path/account_id/node_key_file for each node
+    #[structopt(long)]
+    pub node_key_file: Option<String>,
```

Comment:
Is there a reason why a user would prefer to specify a `node_key_file` different from `p2p_secret`? Maybe we should always use the defailt value?

Reply:
devops guys might need - better safe than sorry IMHO